server/eclipse-project/src/main/java/us/freeandfair/corla/model/AuditBoardDashboard.java

```diff
@@ -179,4 +192,14 @@ public boolean submitAuditCVR(final CastVoteRecord the_cvr_under_audit,
 
     return result;
   }
+  
+  /**
+   * @return the list of audit CVRs submitted. The result will contain
+   * a null value for each element in the cvrsToAudit() list where there
+   * has been no audit CVR submitted. Thus, most computations on this list
+   * will use the sublist preceding the first null value.
+   */
+  public List<Long> submittedAuditCVRs() {
+    return Collections.unmodifiableList(my_submitted_audit_cvrs);
```

Comment:
Nice.  I learned something new.  I haven't been using immutability like this on references that have been leaking through APIs.  We should address that in one go later one.  Another issue to get filed.